feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go

```diff
@@ -312,33 +312,83 @@ func removeClient(handle *p4rt_client.P4RTClient) {
 	handle.ServerDisconnect()
 }
 
-// Test Zero client with 0 electionID
-func TestZeroMaster(t *testing.T) {
+// Test client with unset electionId
+func TestUnsetElectionid(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 	configureDeviceId(t, dut)
 	configurePortId(t, dut)
-	test := testArgs{
-		desc:       pDesc,
-		lowID:      inId0,
-		highID:     inId0,
-		handle:     clientConnection(t, dut),
-		deviceID:   deviceId,
-		wantFail:   true,
-		wantStatus: 3,
+	clients := []testArgs{
+		{desc: pDesc,
+			handle:     clientConnection(t, dut),
+			deviceID:   deviceId,
+			wantStatus: 5,
+		},
+		{desc: sDesc,
+			handle:     clientConnection(t, dut),
+			deviceID:   deviceId,
+			wantStatus: 5,
```

Comment:
Can you use `codes.NotFound` which is internally a uint32 instead of hardcoding to `5`.

This will make the test expectations clearer.

https://pkg.go.dev/google.golang.org/grpc@v1.53.0/codes

Reply:
Updated with codes.NotFound instead of hardcoded value.